### ISSUE TITLE
`onedark#extend_highlight` doesn't extend `cterm` options

### ISSUE THREAD

**Issue by randallwc** [joshdick/onedark.vim#301]
I think here is the issue where the changes to `cterm` are just thrown out.

https://github.com/joshdick/onedark.vim/blob/7db2ed5b825a311d0f6d12694d4738cf60106dc8/colors/onedark.vim#L78-L87

This can be seen if you try

## this doesn't work
```vim
augroup colorextend
    autocmd!
    autocmd ColorScheme * call onedark#extend_highlight("Search", { "cterm": "bold,italic" })
augroup END
```

## this works
```vim
augroup colorextend
    autocmd!
    autocmd ColorScheme * call onedark#set_highlight("Search", { "cterm": "bold,italic" })
augroup END
```

> aside:
> I also think `term` options are also thrown out by looking at the code, but I did not test this hypothesis.

**Comment by joshdick** [comment]
This issue was fixed by #307, but I'm curious about:

> aside:
> I also think `term` options are also thrown out by looking at the code, but I did not test this hypothesis.

Can you clarify what you mean by `term` options?

**Comment by randallwc** [comment]
`:h highlight-term`
